index.bs

```diff
@@ -2403,22 +2436,24 @@ When this method is invoked, the [=authenticator=] MUST perform the following pr
 1. If any error occurred while generating the [=assertion signature=], return an error code equivalent to "{{UnknownError}}" and
     terminate the operation.
 
+<!-- Note: this next step is actually a top-level step, but bikeshed wanted it indented this much in order to compile w/o errors and to render properly.
+-->
     <li id='authenticatorGetAssertion-return-values'>
-    Return to the user agent:
-        - |selectedCredential|'s [=credential ID=], if either a [=list=] of credentials of [=list/size=] 2 or greater was supplied
-            by the client, or no such [=list=] was supplied. Otherwise, return only the below values.
-
-            Note: If the client supplies a [=list=] of exactly one credential and it was successfully employed, then its
-                [=credential ID=] is not returned since the client already knows it. This saves transmitting these bytes over
-                what may be a constrained connection in what is likely a common case.
-
-        - |authenticatorData|
-        - |signature|
-        - The [=user handle=] associated with |selectedCredential|, if available.
+        Return to the user agent:
+            - |selectedCredential|.[=public key credential source/id=], if either a list of credentials of length 2 or greater was
+                supplied by the client, or no such list was supplied.
+    
+                    Note: If the client supplies a [=list=] of exactly one credential and it was successfully employed, then its
+                        [=credential ID=] is not returned since the client already knows it. This saves transmitting these bytes over
+                        what may be a constrained connection in what is likely a common case.
+    
+            - |authenticatorData|
+            - |signature|
+            - |selectedCredential|.[=public key credential source/userHandle=]
```

Comment:
CTAP does not store the user handle for first factor credentials, so it seems misleading to imply that it must always be returned (unless we point out at [the definition site](https://pr-preview.s3.amazonaws.com/jyasskin/webauthn/pull/623.html#ref-for-user-handle%E2%91%A0) that the `userHandle` field is nullable). See also the client's [getAssertion](https://w3c.github.io/webauthn/#ref-for-assertioncreationdata-userhandleresult) method.